tests/tests/swfs/avm2/stage3d_sampler_partial_upload/test.toml

```diff
@@ -0,0 +1,8 @@
+num_frames = 1
+
+[image_comparisons.output]
+tolerance = 1
+max_outliers = 782
```

Comment:
This is pretty high- can this be decreased by increasing `tolerance`?

Reply:
I'll try this in a follow up